docs/config.c

```diff
@@ -3785,14 +3785,15 @@
 ** to save attachments to files named like:
 ** .ts
 ** =?iso-8859-1?Q?file=5F=E4=5F991116=2Ezip?=
+** =?utf-8?Q?z=C4=99ta.png?=
 ** .te
 ** .pp
 ** When this variable is \fIset\fP interactively, the change won't be
 ** active until you change folders.
 ** .pp
 ** Note that this use of RFC2047's encoding is explicitly
 ** prohibited by the standard, but nevertheless encountered in the
-** wild.
+** wild and produced by, i.a., Outlook.
```

Comment:
```suggestion
** wild and produced by, e.g., Outlook.
```

"i.a." inter alia isn't common in English, but "e.g." exempli gratia is.

Reply:
Fair enough, updated.